### ISSUE TITLE
Boot Crash

### ISSUE THREAD

**Issue by Cyronix86** [aleab/toastify#89]
## EXPECTED BEHAVIOUR
Start Toastify > Toastify starts Spotify

## ACTUAL BEHAVIOUR
Error Box
[Toastify.log](https://github.com/aleab/toastify/files/2282428/Toastify.log)
![untitled](https://user-images.githubusercontent.com/42339851/44023654-ec450346-9ee2-11e8-9f93-c78cd364c495.png)

**Comment by aleab** [comment]
Apparently Toastify can't find Spotify. Is it installed correctly (Spotify)? Which version do you have installed? The desktop version or the one from the Windows store?

If you have the desktop version can you execute the following command in the Windows command prompt (cmd) and see what it returns?

```cmd
reg query HKCU\Software\Microsoft\Windows\CurrentVersion\Uninstall\Spotify /v InstallLocation
```

**Comment by WegPast** [comment]
Same problem here, I have the desktop version of spotify.
Spotify is correctly installed.

Your command return me that the key doesn't exist.

**Comment by Cyronix86** [comment]
System was unable to find reg key

**Comment by Cyronix86** [comment]
Seems I have the app store spotify

**Comment by Cyronix86** [comment]
Yeah toast works fine once spot is already running, I'll try a fresh desktop version install.

**Comment by aleab** [comment]
@WegPast @Cyronix86 

That's odd! 🤔
What about these commands?
```cmd
echo off
for %r in (HKLM\SOFTWARE HKLM\SOFTWARE\WOW6432Node HKCU\Software) do (
  echo.
  echo %r\Microsoft\Windows\CurrentVersion\Uninstall\Spotify
  reg query "%r\Microsoft\Windows\CurrentVersion\Uninstall\Spotify" /s
)
```

**Comment by Cyronix86** [comment]
![image](https://user-images.githubusercontent.com/42339851/44030683-78de033c-9ef9-11e8-8c45-dda396f577cc.png)

**Comment by Cyronix86** [comment]
![image](https://user-images.githubusercontent.com/42339851/44030744-a36290e6-9ef9-11e8-8f66-c3ffea452d98.png)

**Comment by aleab** [comment]
I know where the problem is. Until I release the fix, starting Spotify first is the only workaround.
I'll keep you posted.

**Comment by Cyronix86** [comment]
Great, thanks!

**Comment by aleab** [comment]
Closing the issue as this has been fixed in [v1.10.10](https://github.com/aleab/toastify/releases/tag/v1.10.10).
